packages/core/src/io/platform-io.ts

```diff
@@ -74,7 +74,8 @@ export abstract class PlatformIO {
 			if (!resource.uri) {
 				const isGLBBuffer = isGLB && index === images.length;
 				if ((resource as GLTF.IImage)['bufferView'] === undefined && !isGLBBuffer) {
-					throw new Error('Missing resource URI.');
+					// TODO(DO NOT SUBMIT): There was a reason this threw an error...
+					console.warn('PlatformIO: Missing resource URI.');
```

Comment:
Reasoning for this error was in https://github.com/donmccurdy/glTF-Transform/issues/178 and https://github.com/donmccurdy/glTF-Transform/pull/179, I guess I'll need to find a clearer way to detect missing resources without knowledge in this module of `EXT_meshopt_compression`.